pkg/controller/servicebindingrequest/binder_test.go

```diff
@@ -480,3 +483,61 @@ func Test_extraFieldsModifier(t *testing.T) {
 	})
 
 }
+
+func TestBindAndUnbind(t *testing.T) {
+
+	t.Run("bind and unbind", func(t *testing.T) {
```

Comment:
can you please share more infos what we are testing here, given the scope of PR? And update the test name accordingly: what is the context and the expected outcome?

Reply:
Previously there is no UT for bind and unbind. I add them and have added more descriptions.